lib/taurus/qt/qtgui/display/test/test_tauruslabel.py

```diff
@@ -94,47 +94,47 @@ def test_relativemodelclass(self):
                                    model='tango:' + DEV_NAME + '/double_scalar')
 
 
-@testOldFgroles(fgRole='value', expected='1.23 mm')
-@testOldFgroles(fgRole='w_value', expected='0.00 mm')
-@testOldFgroles(fgRole='state', expected='Ready')
+# @testOldFgroles(fgRole='value', expected='1.23 mm')
```

Comment:
I guess you commented out these tests while debugging but you forgot to uncoment them back....

Reply:
You are right, I forget to remove the "kk" commit.